themes/powerline/powerline.theme.bash

```diff
@@ -37,13 +37,17 @@ function set_rgb_color {
 }
 
 function powerline_shell_prompt {
+    SHELL_PROMPT_COLOR=${SHELL_THEME_PROMPT_COLOR}
+    if sudo -n true &>/dev/null; then
```

Comment:
OK, I'll change that, was wondering why uptime was used lol.
Does `grep -q` work on OSX?